Source/WTF/wtf/HashTable.h

```diff
@@ -901,8 +896,10 @@ DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(HashTable);
             
                 if (isDeletedBucket(*entry))
                     deletedEntry = entry;
-                else if (HashTranslator::equal(Extractor::extract(*entry), key))
+                else if (HashTranslator::equal(Extractor::extract(*entry), key)) {
+                    validateKey<ShouldValidateKey::No>(*entry);
```

Comment:
I think it would be clearer to have a non-template function to use in a case like this. It’s so wordy to pass the template argument.